src/backend/common/helpers/insights_helper.py

```diff
@@ -1088,4 +1151,48 @@ def doOverallAwardInsights(self) -> List[Insight]:
                 )
             )
 
+        insights.extend(self.do_overall_leaderboard_award_insights())
+
         return insights
+
+    @classmethod
+    def do_overall_leaderboard_award_insights(cls) -> List[Insight]:
+        overall_insights = []
+        for insight_type in Insight.TYPED_LEADERBOARD_AWARD_INSIGHTS:
+            insights = Insight.query(
+                Insight.name == Insight.INSIGHT_NAMES[insight_type],
+                Insight.year != 0,
+            ).fetch(1000)
+
+            data = defaultdict(int)
+            for insight in insights:
+                for leaderboard_ranking in insight.data:
+                    for team in leaderboard_ranking["team_keys"]:
+                        data[team] += leaderboard_ranking["value"]
+
+            overall_insights.append(
+                cls._create_leaderboard_from_dict_counts(data, insight_type, year=0)
+            )
+
+        return overall_insights
+
+    @classmethod
+    def do_overall_leaderboard_match_insights(cls) -> List[Insight]:
```

Comment:
Can these two methods be combined taking in a list of insight names as an argument?